MAINTAINERS.md

```diff
@@ -0,0 +1,12 @@
+This page lists all active maintainers of this repository.
+
+See [CONTRIBUTING.md](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/CONTRIBUTING.md)
+for general contribution guidelines.
+
+## Maintainers (in alphabetical order)
+  
+- [Evan Cordell](github.com/ecordell), Authzed
+- [Nick Hale](github.com/njhale), Red Hat
+- [Kevin Rizza](github.com/kevinrizza), Red Hat
+- [Alexander Greene](github.com/awgreene), Red Hat
+- [Vu Dinh](github.com/dinhxuanvu), Red Hat
```

Comment:
> (in alphabetical order)

This doesn't look like that's the case looking at this list?

Reply:
oops